src/main/java/legacy/org/postgresql/Driver.java

```diff
@@ -497,7 +503,7 @@ public DriverPropertyInfo[] getPropertyInfo(String url, Properties info) throws
         return props;
     }
 
-    public static final int MAJORVERSION = @MAJORVERSION@;
+    public static final int MAJORVERSION = 9;
```

Comment:
Hardcoded these version number. It used to be using ANT to automatically search and replace values in the code via ANT config file. There's really no need to automate this.